Tests/InputFilterTest.php

```diff
@@ -474,6 +474,18 @@ public function casesGeneric()
 				'C:\Documents\Newsletters\Summer2018.pdf',
 				'From generic cases'
 			),
+			'windows path with /'                                           => array(
+				'path',
+				'C:\\Documents\\Newsletters/tmp',
+				'C:\Documents\Newsletters/tmp',
```

Comment:
I would expect 'C:\Documents\Newsletters\tmp'

Reply:
Then the `cleanPath` has to be changed to replace the slashes.

Reply:
Otherwise the test will fail if I change it only in the test here.

Reply:
Yes